mmseg/utils/find_latest_checkpoint.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import glob
+import os.path as osp
+import warnings
+
+
+def find_latest_checkpoint(path, suffix='pth'):
+    """Find the latest checkpoint from the working directory.
+
```

Comment:
This function is for finding the latest checkpoint, which will be used when automatically resume, modified from....